packages/lesswrong/server/emails/inactiveUserSummaryCron.tsx

```diff
@@ -0,0 +1,176 @@
+import React from "react";
+import { hasInactiveSummaryEmail } from "@/lib/betas"
+import { generateEmail, wrapAndSendEmail } from "./renderEmail";
+import { fetchFragment } from "../fetchFragment";
+import {
+  type BestReaction,
+  EmailInactiveUserSummary,
+} from "../emailComponents/EmailInactiveUserSummary";
+import { EmailWrapper } from "../emailComponents/EmailWrapper";
+import FeatureStrategy from "../recommendations/FeatureStrategy";
+import Notifications from "../collections/notifications/collection";
+import Users from "../collections/users/collection";
+import UsersRepo from "../repos/UsersRepo";
+import VotesRepo from "../repos/VotesRepo";
+import PostsRepo from "../repos/PostsRepo";
+import sum from "lodash/sum";
+import { addCronJob } from "../cron/cronUtil";
+
+const chooseBestReaction = (
+  reactions: Record<string, number>,
+): BestReaction | undefined => {
+  const entries = Object.entries(reactions);
+  const maxEntry = entries.reduce(
+    (max, entry) => entry[1] > max[1] ? entry : max,
+    entries[0],
+  );
+  return maxEntry?.[1] ? {name: maxEntry[0], count: maxEntry[1]} : undefined;
+}
+
+const sendInactiveUserSummaryEmail = async (
+  {fetchActivitySince, ...user}: DbUser & {fetchActivitySince: Date},
+  dryRun = false,
+) => {
+  if (!hasInactiveSummaryEmail || !user.email) {
+    return;
+  }
+
+  const now = new Date();
+  const votesRepo = new VotesRepo();
+  const postsRepo = new PostsRepo();
+  const featureStrategy = new FeatureStrategy();
+
+  const [
+    karmaChanges,
+    reacts,
+    agreements,
+    mostCommentedPost,
+    unreadNotifications,
+    recommendations,
+  ] = await Promise.all([
+    votesRepo.getEAKarmaChanges({
+      userId: user._id,
+      startDate: fetchActivitySince,
+      endDate: now,
+      showNegative: true,
+    }),
+    votesRepo.getEAReactsReceived(user._id, fetchActivitySince, now),
+    votesRepo.getEAAgreements(user._id, fetchActivitySince, now),
+    postsRepo.getUsersMostCommentedPostSince(user._id, fetchActivitySince),
+    Notifications.find({
+      userId: user._id,
+      createdAt: {$gt: fetchActivitySince},
+      viewed: false,
+      emailed: false,
+      deleted: false,
+    }).count(),
+    featureStrategy.recommend(user, 5, {
+      name: "feature",
+      postId: "",
+      features: [
+        {feature: "subscribedAuthorPosts", weight: 1},
+        {feature: "subscribedTagPosts", weight: 0.9},
+        {feature: "curated", weight: 0.75},
+        {feature: "karma", weight: 0.4},
+      ],
+    }, {
+      publishedAfter: fetchActivitySince,
```

Comment:
FYI I noticed that this compares the date to `createdAt` but it should be comparing it to `postedAt`